Seq.App.HipChat/HipChatReactor.cs

```diff
@@ -52,7 +60,7 @@ public class HipChatReactor : Reactor, ISubscribeTo<LogEventData>
         {
             using (var client = new HttpClient())
             {
-                client.BaseAddress = new Uri("https://api.hipchat.com/v2/");
+                client.BaseAddress = new Uri(HipChatBaseUrl ?? DefaultHipChatBaseUrl);
```

Comment:
Thanks for your contrib!

I'm unsure about Seq's behaviour, but it would feel more safe using string.IsNullOrWhitespace to check for empty value.

Check the following comment for how to run locally: https://github.com/stayhard/Seq.App.HipChat/pull/1#issuecomment-116478104